frontend/src/pages/AdminPage/AdminPage.js

```diff
@@ -36,21 +37,22 @@ function AdminPage() {
   const { isShowing, toggle } = useModal();
 
   return (
-    <div>
-      <h1 className="main-title">Ticket Types</h1>
-      {fakeData.map(adminTicket => (
-        <Ticket key={adminTicket.id} {...adminTicket}>
-          <Button onClick={toggle}>DELETE</Button>
-        </Ticket>
-      ))}
-      <Modal hide={toggle} isShowing={isShowing}>
-        <Button onClick={toggle}>OK</Button>
-        <Button buttonStyle={'btn--danger--solid--btn'} onClick={toggle}>
-          Cancel
-        </Button>
-        Are you sure?
-      </Modal>
-    </div>
+      <div>
+        <Header type="admin" />
```

Comment:
@tamas-w ah, I see what you tried to do here.
but that's not really going to be good. The header should be placed actually in the `App` component and it should work differently based on what's in the redux store / localstorage.